parlai/scripts/train_model.py

```diff
@@ -107,32 +110,44 @@ def setup_args(parser=None):
     return parser
 
 
-def run_eval(agent, opt, datatype, max_exs=-1, write_log=False, valid_world=None):
-    """Eval on validation/test data.
-    - Agent is the agent to use for the evaluation.
-    - opt is the options that specific the task, eval_task, etc
-    - datatype is the datatype to use, such as "valid" or "test"
-    - write_log specifies to write metrics to file if the model_file is set
-    - max_exs limits the number of examples if max_exs > 0
-    - valid_world can be an existing world which will be reset instead of reinitialized
-    """
-    print('[ running eval: ' + datatype + ' ]')
+def _maybe_load_world(agent, opt, datatype):
+    if not is_primary_worker():
```

Comment:
this was a bit confusing without the rest of the context. maybe call it `_maybe_load_eval_world`?

Reply:
(y)